client/verta/verta/_cli/remote.py

```diff
@@ -14,11 +16,26 @@ def remote():
 @click.argument('url')
 def add(name, url):
     """Add a remote repository."""
-    click.echo("registered repo {} at url {}".format(name, url))
+    # TODO: validate `url`
+
+    with _config_utils.read_config() as config:
+        if name in config.get('remotes', {}):
+            raise click.BadParameter("remote '{}' already exists".format(name))
+
+    with _config_utils.write_config() as config:
+        config.setdefault('remotes', {})[name] = {
+            'url': url,
+            'branch': "master",
+        }
 
 
 @remote.command()
 @click.argument('name')
 def use(name):
     """Set a remote repository as the active one."""
-    click.echo("now using repo {}".format(name))
+    with _config_utils.read_config() as config:
+        if name not in config.get('remotes', {}):
+            raise click.BadParameter("no such remote: '{}'".format(name))
+
+    with _config_utils.write_config() as config:
+        config['current-remote'] = name
```

Comment:
We should set the remote added with `add` if it's the only one.

Reply:
Great thinking! 0382b7c